controller/controller_tool/tool/src/py/Controller.py

```diff
@@ -128,67 +127,67 @@ def is_valid_dds_domain(self, dds_domain):
     def init_dds(self, dds_domain):
         """Initialize DDS entities."""
         factory = fastdds.DomainParticipantFactory.get_instance()
-        participant_qos = fastdds.DomainParticipantQos()
-        factory.get_default_participant_qos(participant_qos)
-        participant_qos.name('DDSRecorderController')
+        self.participant_qos = fastdds.DomainParticipantQos()
```

Comment:
why to store qos in a class variable?

Reply:
Some of these needed to be stored in object, I did not test exactly which ones. But indeed, in particular this one is not required.